### PR TITLE
fix: convert bf to bold(upright(..))

### LINKED ISSUE #174: [Bug] convert `\bf` wrongly
`\bf{..}` should be `bold(upright(..))`, but mitex convert it to `bold(..)`.

### PR DESCRIPTION
**Pull request by Chengyuan-artist**
fix #174 

before:
image

after:
image

### CODE REVIEW

File: packages/mitex/specs/latex/standard.typ [hunk 1]
```diff
@@ -146,7 +146,7 @@
   textstyle: greedy-handle("mitexinline", math.inline),
   scriptstyle: greedy-handle("mitexscript", math.script),
   scriptscriptstyle: greedy-handle("mitexsscript", math.sscript),
-  bf: greedy-handle("mitexbold", math.bold),
+  bf: define-cmd(1, alias: "mitexbold", handle: it => math.bold(math.upright(it))),
```
Comment (OrangeX4) [inline packages/mitex/specs/latex/standard.typ:149]: Thanks. Can't we continue to use `greedy-handle`? Does `greedy-handle("mitexbf", it = math.bold(math.upright(it)))` work?

**Comment by OrangeX4** [comment]
I guess CI needs some maintenance...

File: packages/mitex/specs/latex/standard.typ (hunk 1 above)
Comment (Chengyuan-artist) [reply packages/mitex/specs/latex/standard.typ:149]: `greedy-handle("mitexbold", it => math.bold(math.upright(it)))`  can also work after a quick try.

Comment (Chengyuan-artist) [reply packages/mitex/specs/latex/standard.typ:149]: I will change this commit before CI is fixed.

**Comment by Chengyuan-artist** [comment]
I fixed the ci related errors.

**Review by Enter-tainer — APPROVED** [review]
Thanks! cc @OrangeX4 PTAL

**Review by Myriad-Dreamin — APPROVED** [review]

**Pull request merged**